test/Main.purs

```diff
@@ -1,14 +1,13 @@
 module Test.Main where
 
-import Prelude
+import Prelude (class Eq, class Show, Unit, bind, const, discard, pure, show, unit, ($), (<$>), (<*>), (<>), (=<<), (>>=))
```

Comment:
Was there a particular reason for this? You can just open import this like it was, PS allows one open import without any warnings.

Reply:
ha I didn't know that. It must have given me an error at some point when I had another import in there.